### PR TITLE
add fhir bundle

### PR DESCRIPTION
**Pull request by lei0zhou**
in this pr: 
- added fhir bundle including Patient, Questionnaire, and ValueSet resource
	- 	ValueSet ref: [Colorado Counties (64 counties) | Colorado Department of Local Affairs](https://dola.colorado.gov/lgis/counties.jsf)
- edited the makefile for loading this bundle
- $populate
	- 	Get request as `[FHIR base url]/Questionnaire/[id]/$populate?subject=[patient_id]` would populate the answer

### CODE REVIEW

File: hapi/R4/CUSTOM/Jane_Doe.json [hunk 1]
```diff
@@ -0,0 +1,438 @@
+{
+    "resourceType": "Bundle",
+    "type": "transaction",
+    "entry": [
+        {
+            "fullUrl": "urn:uuid:69b3b293-17df-4932-8a6a-d2dd5b79cd81",
+            "resource": {
+                "resourceType": "ValueSet",
+                "id": "county_valueset",
```
Comment (jc00ke) [inline hapi/R4/CUSTOM/Jane_Doe.json]: Let's rename this to be a little more specific: `colorado_counties`

File: hapi/R4/CUSTOM/Jane_Doe.json [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+                        "required": true,
+                        "readOnly": true
+                    },
+                    {
+                        "extension": [
+                            {
+                                "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-initialExpression",
+                                "valueExpression": {
+                                    "language": "text/fhirpath",
+                                    "expression": "%subject.address.district"
```
Comment (jc00ke) [inline hapi/R4/CUSTOM/Jane_Doe.json:420]: :raised_hands:

**Review by jc00ke — CHANGES_REQUESTED** [review]

File: hapi/R4/CUSTOM/Jane_Doe.json [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+                                }
+                            }
+                        ],
+                        "linkId": "patient.county",
+                        "definition": "Patient.address.county",
+                        "text": "County",
+                        "type": "choice",
+                        "required": true,
+                        "answerValueSet": "https://dola.colorado.gov/lgis/counties.jsf"
```
Comment (jc00ke) [inline hapi/R4/CUSTOM/Jane_Doe.json:429]: :+1:

File: hapi/R4/CUSTOM/Jane_Doe.json (hunk 1 above)
Comment (lei0zhou) [reply hapi/R4/CUSTOM/Jane_Doe.json]: updated

**Review by jc00ke — APPROVED** [review]

**Pull request merged**